packages/extension-shared/src/util/isHexString.ts

```diff
@@ -0,0 +1,7 @@
+export type HexString = `0x${string}`
+
+export const REGEX_HEX_STRING = /^0x[0-9a-fA-F]*$/
```

Comment:
Suggestion to make `isHexString` return `false` when the input is an empty string, a la `""`

```suggestion
export const REGEX_HEX_STRING = /^0x[0-9a-fA-F]+$/
```